crates/re_renderer/src/mesh.rs

```diff
@@ -251,11 +338,17 @@ impl GpuMesh {
             materials
         };
 
+        let vb_colors_start = vb_positions_size;
+        let vb_normals_start = vb_colors_start + vb_color_size;
+        let vb_texcoord_start = vb_normals_start + vb_normals_size;
```

Comment:
[not actionable] I wonder if `staging_buffer.extend_from_slice` should return these ranges as a convenience feature. What do you think? Helpful or too weird for an `extend_from_slice` method? It's technically also different buffer ofc.

Reply:
This is tedious and error prone, so some helper would be nice